ExomeDepth/make_BEDdetail.py

```diff
@@ -0,0 +1,300 @@
+            bfs_parent_list, ratios_parent_list = zip(*sorted(zip(bfs_parent_list,ratios_parent_list)))
+            bfs_parent = ' '.join(map(str, bfs_parent_list))
+            ratios_parent = ' '.join(map(str, ratios_parent_list))
+            
+            average_correlation_parent = "%.3f" % (float(statistics.mean(correlation_parent)))
+            average_deldupratio_parent = "%.0f" % (float(statistics.mean(deldupratio_parent)))
+            average_totalcalls_parent = "%.0f" % (float(statistics.mean(totalcalls_parent)))
+            average_bf_parent = "%.1f" % (float(statistics.mean(bf_parent)))
+            average_ratio_parent = "%.2f" % (float(statistics.mean(ratio_parent)))
+            
+            if totalcount_parent > 1:
+                stdev_correlation_parent = "%.3f" % (float(statistics.stdev(correlation_parent)))
+                stdev_deldupratio_parent = "%.0f" % (float(statistics.stdev(deldupratio_parent)))
+                stdev_totalcalls_parent = "%.0f" % (float(statistics.stdev(totalcalls_parent)))
+                stdev_bf_parent = "%.1f" % (float(statistics.stdev(bf_parent)))
+                stdev_ratio_parent = "%.2f" % (float(statistics.stdev(ratio_parent)))
+
+        template_file = Template(open(settings.html).read())
+        substitute_dic = {'total_child' : totalcount_child, 
+            'cr_avr_child' : average_correlation_child, 'cr_stdev_child' : stdev_correlation_child,
+            'deldup_avr_child': average_deldupratio_child, 'deldup_stdev_child' : stdev_deldupratio_child, 
+            'numbercalls_avr_child' : average_totalcalls_child, 'numbercalls_stdev_child' : stdev_totalcalls_child,
+            'total_parent' : totalcount_parent, 
+            'cr_avr_parent' : average_correlation_parent, 'cr_stdev_parent' : stdev_correlation_parent, 
+            'deldup_avr_parent': average_deldupratio_parent, 'deldup_stdev_parent' : stdev_deldupratio_parent, 
+            'numbercalls_avr_parent' : average_totalcalls_parent, 'numbercalls_stdev_parent' : stdev_totalcalls_parent,
+            'average_bf_child': average_bf_child,  'stdev_bf_child': stdev_bf_child,
+            'average_ratio_child': average_ratio_child, 'stdev_ratio_child': stdev_ratio_child,
+            'average_bf_parent': average_bf_parent, 'stdev_bf_parent': stdev_bf_parent,
+            'average_ratio_parent': average_ratio_parent, 'stdev_ratio_parent': stdev_ratio_parent,
+            'bfs_child' : bfs_child, 'ratios_child': ratios_child,
+            'bfs_parent' : bfs_parent, 'ratios_parent': ratios_parent
+        }
+        new_file = template_file.substitute(substitute_dic)       
+        event_list.append("".join(new_file.split("\n")))
+        total_event_list.append(event_list)
+    total_event_list.sort(key=lambda x:(settings.chromosome_order[x[0]], int(x[1])))
+      
+    for event in total_event_list:
+        event[0] = "chr{original}".format(original=event[0])
+        joined_event = "\t".join([str(i) for i in event])
```

Comment:
Dit is al een str?

Reply:
Nope. De elementen in event zijn ook int (bv start). Denk dat het handiger is om ze hier een ander type te geven dan bij het vullen van de events.